### DISCUSSION TITLE
Disable docbot for the pulsar-site repository

### DISCUSSION THREAD

**Discussion by visortelle** [apache/pulsar#22135] [Ideas]
Hi :wave:
[Docbot](https://github.com/apache/pulsar-test-infra/tree/master/docbot) automatically label pull requests based on the checked task list.

<img width="748" alt="Screenshot 2024-02-27 at 6 32 20 PM" src="https://github.com/apache/pulsar/assets/9302460/b6a845c2-ca58-41f2-b4d9-68a0764573a3">

I wonder, does the "docbot" bring any value to the https://github.com/apache/pulsar-site repo?

The content of this repo is mostly docs itself. 

PRs don’t implement any features that require docs. If they are, it's rather an exception.

Maybe I'm missing something.

cc @lhotari @tisonkun

**Comment by lhotari** [comment]
+1000

**Reply by visortelle** [reply]
OK. Then I'll prepare a PR for this a bit later. 🙂
